boudicca.base/publisher-event-html/src/main/kotlin/base/boudicca/publisher/event/html/PublisherHtmlConfiguration.kt

```diff
@@ -27,13 +30,15 @@ class PublisherHtmlConfiguration(private val properties: PublisherHtmlProperties
         titleValueResolver: TitleValueResolver
     ): ViewResolver {
         val viewResolver = HandlebarsViewResolver()
-        viewResolver.order = 0 //we have to decrease the order so ours is first (default is Int.MAX_VALUE)
+        viewResolver.order = Ordered.HIGHEST_PRECEDENCE
```

Comment:
please leave it at 0 or `Ordered.HIGHEST_PRECEDENCE + 1` since we need to make sure our order is after the ContentNegotiatingViewResolver